File: Backend/TravellifeChaser/Models/AirlinesSystem/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection.Metadata.Ecma335;
using System.Threading.Tasks;

namespace TravellifeChaser.Models
{
    public class Address
    {
        public int Id { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public double? Longitude { get; set; }
        public double? Latitude { get; set; }

        public ICollection<Airline> Airlines { get; set; } = new List<Airline>();// 1 - N Airline - Address
        public ICollection<Airport> Airports { get; set; } = new List<Airport>();
        public ICollection<User> Users { get; set; } = new List<User>();
    }
}

File: AvioAndRentACarSystem/src/app/AirlineModel/address.ts
export class Address
{
    id: number;
    city: string;
    country: string;
    longitude: number;
    latitude: number;

    constructor(id: number, city: string, country: string, longitude?: number, latitude?: number)
    {
        this.id = id;
        this.city = city;
        this.country = country;
        this.longitude = longitude;
        this.latitude = latitude; 
    }
}